patches/content-browser-service_worker-service_worker_version.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/content/browser/service_worker/service_worker_version.cc b/content/browser/service_worker/service_worker_version.cc
+index 26cdf768f189f054855f351ebfbb427aec049c01..b102140e0501335983d110bbf8080efc48baedd6 100644
+--- a/content/browser/service_worker/service_worker_version.cc
++++ b/content/browser/service_worker/service_worker_version.cc
+@@ -74,7 +74,7 @@ const base::FeatureParam<int> kUpdateDelayParam{
+ // timeout.
+ const base::FeatureParam<int> kTerminationDelayParam{
```

Comment:
this patch can be avoided. You can put `kTerminationDelayParam` as a member of `ServiceWorkerVersion` class and add something to touch `::content::kTerminationDelayParam.Get()` to hide the unused warning.

Reply:
It is really preferable to one-line patch?)

Reply:
well, I think for constants changes it makes sense to keep things simple and we can actually go with single-line git patches.
@mkarolin @mariospr wdyt? are you okay with that?